README.md

```diff
@@ -190,3 +190,33 @@ When given a command, Open Crawler will run until the process is finished.
+| **Interface**                                                                                                                                          | CLI          | GUI (Kibana)            |
+| **Hosting**                                                                                                                                            | Self-hosted  | Elastic Cloud           |
+| **State management**                                                                                                                                   | Stateless    | Elasticsearch indices   |
+| **Compatible with [Elasticsearch Serverless](https://www.elastic.co/docs/current/serverless)**                                                         | Yes          | No                      |
+| **Unrestricted index naming**                                                                                                                          | Yes          | No                      |
+| **Indexing using `_bulk` API**                                                                                                                         | Yes          | No                      |
+| **[Ingest pipelines](./docs/features/INGEST_PIPELINES.md)**                                                                                            | Yes          | Yes                     |
+| **[Binary content extraction](./docs/features/BINARY_CONTENT_EXTRACTION.md)**                                                                          | Yes          | Yes                     |
+| **[Crawler directives](./docs/features/CRAWLER_DIRECTIVES.md)**                                                                                        | Yes *        | Yes                     |
+| **[Scheduling](#scheduling-recurring-crawl-jobs)**                                                                                                     | Yes          | Yes                     |
+| **Crawler results history and metadata**                                                                                                               | No **        | Yes                     |
+| **[Duplicate content handling](https://www.elastic.co/guide/en/enterprise-search/current/crawler-managing.html#crawler-managing-duplicate-documents)** | No           | Yes                     |
+| **Event Logging**                                                                                                                                      | No ***       | Yes                     |
+| **Monitoring**                                                                                                                                         | No           | Yes                     |
+
+*\* Open Crawler does not support [meta tags and data attributes to extract custom fields](https://www.elastic.co/guide/en/enterprise-search/current/crawler-content.html#crawler-content-meta-tags-content-extraction).*\
```

Comment:
Is this on the roadmap though? Is it worth signaling like you had in the blog which features are planned/scheduled even if they're not available _yet_?

Reply:
It is on the roadmap for GA / v0.3. I'll add a mention to that (and others if they're on the roadmap).